miners/text/gpt2_exodus.py

```diff
@@ -133,6 +134,28 @@ def config() -> 'bittensor.Config':
         bittensor.axon.add_args( parser )
         GPT2Nucleus.add_args( parser )
         SGMOERouter.add_args( parser )
+                
+        
+        
+        config_file_path = vars(parser.parse_known_args()[0])['neuron.config']
+        if config_file_path:
+            #loads config_file and updates defaults
+            
+            config_file_path = os.path.expanduser(config_file_path)
+
+                
+            try:
+                with open(config_file_path) as f:
+                    params_config = yaml.safe_load(f)
+                    
+                    print('Config File Detected at' ,config_file_path, ', updating defaults')
+                    parser.set_defaults(**params_config)
+                    
+            except Exception as E:
+                print('Error in loading:', E, ', using default parser settings')
```

Comment:
Looks great @Eugene-hu , but can we switch the printing formatting? Throughout the code we use the `'{}'.format()` format. For example, `print("Error in loading {} using default parser settings".format(e))`. It's more python-3 :) 

Also, let's avoid using capital letters, best use `Exception as e`.

Reply:
Fixed!